Tests/CryptorRSATests/CryptorRSATests.swift

```diff
@@ -80,7 +80,7 @@ class CryptorRSATests: XCTestCase {
 		
         let path = CryptorRSATests.getFilePath(for: "public", ofType: "der")
         XCTAssertNotNil(path)
-        
+		
```

Comment:
Can this white space (and the rest of the similar lines) just be deleted?

Reply:
BlueRSA uses a mix of both tabs and spaces through the project (Even switching within the same function) which caused lots of white space changes. I have removed most of those and will pick up the last few here. 
I won't change the sha algorithms in the tests since they have varying amounts of tabs and then spaces within the same line and i can't bring myself to replicate this.